src/Utils/OrientationUtils.js

```diff
@@ -211,6 +264,54 @@ export default {
         };
     },
 
+    /**
+     * Set the quaternion to correct for the
```

Comment:
same documentation that `quaternionFromTMerc`

Reply:
done